scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -419,9 +419,22 @@ abstract class FixedPathSource(path: String*) extends FileSource {
   def localPaths = path.toList
   def hdfsPaths = path.toList
 
+  override def hdfsWritePath = stripTrailingStar(super.hdfsWritePath)
+
   override def toString = getClass.getName + path
   override def hashCode = toString.hashCode
   override def equals(that: Any): Boolean = (that != null) && (that.toString == toString)
+
+  /**
+    * Strip trailing '*' from the path string.
+    */
```

Comment:
@johnynek I guess you're referring to adding a link to that method in the source? If yes, I can make that change. 

I can make this code consistent with TimePathedSource and also drop the `/`.